schemas/workflow.py

```diff
@@ -35,6 +36,12 @@ class WorkflowTaskCreate(_WorkflowTaskBase):
     task_id: int
     workflow_id: Optional[int]
 
+    @validator("task_id", "workflow_id")
+    def positive(cls, _id):
+        if (_id is not None) and _id < 1:
```

Comment:
Dumb question: do IDs always start at 1?

Reply:
@mfranzon ?

Reply:
by default sql AUTOINCREMENT implementation starts from 1, so I think we are safe